### DISCUSSION TITLE
Stream load方式导入数据丢失问题

### DISCUSSION THREAD

**Discussion by trustTiMo** [apache/doris#14779] [A - General / Q&A]
通过 Stream load 方式导入数据时，比如1000行数据，当第500行数据出现异常数据导致错误后，后面的数据不再进行导入，后面500条正常数据丢失，需要怎样解决。目前已经配置max_filter_ratio为1。谢谢

**Comment by smallhibiscus** [comment]
if you set max_filter_ratio = 1,the batch exception data will lost. I understand that it is still necessary to set max_filter_ratio=0, and then solve the error, so that it will not cause data loss.

**Reply by trustTiMo** [reply]
Thank You。但是我的业务逻辑是这种错误数据直接忽略，直接跳过就好。想知道有没有参数设置可以直接跳过错误行。谢谢

**Reply by LemonLiTree** [reply]
添加参数：-H "strict_mode:true" 开启严格模式

**Reply by dataalive** [reply]
开启了严格模式后反而会达不到他丢失错误数据的效果把

**Reply by LemonLiTree** [reply]
> 开启了严格模式后反而会达不到他丢失错误数据的效果把

开启严格模式+容错率参数就是实现错误数据过滤

**Reply by trustTiMo** [reply]
我开启了严格模式，但是仍旧丢失后面的数据。另外读取的文件都是标准格式化的文件，不知道为什么会偶发性出现读取行文件异常而导致后面数据报错。

**Reply by LemonLiTree** [reply]
> 我开启了严格模式，但是仍旧丢失后面的数据。另外读取的文件都是标准格式化的文件，不知道为什么会偶发性出现读取行文件异常而导致后面数据报错。

我比较好奇为什么会导致后面的报错，你能发下表schema 和导入遇到的错误数据样例吗？
